.gitignore

```diff
@@ -1,5 +1,5 @@
 node_modules
 npm-debug.log
-secrets.js
+secrets*.js
```

Comment:
In case you're curious, I added this so that I could have a secrets.local.js as well as secrets.js.

To my secrets.js I added
```
try {
    // try to load in any local config overrides
    let localSecrets = require('./secrets.local.js');
    secrets = Object.assign(secrets, localSecrets);
} catch(err) {}
```
This way I can override stuff locally, like for example the app_port setting so it doesn't conflict with other apps I may be running. Just in case that's helpful to anyone else...